SU-SWS/saa_alumni
# Release v4.0.0

# READY FOR REVIEW

# Summary
- Membership Forms
- Fullprofile API Refactor
- T/S Date Picker Styling

# Review By (Date)
- EOD Thursday, 5/18 (I plan to merge this in Friday morning @ 9AM after getting confirmation that the API deployments are GTG. ETA for the API deployment is Thursday evening)

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to [`membership/join`](https://deploy-preview-668--stanford-alumni.netlify.app/membership/join)
2. Test the following form flows:
- Select `Myself` > From the payment options, select `One-time Payment` > Continue to form > Fill out form > Submit
- Select `Myself` > From the payment options, select `Installments` > Continue to form > Fill out form > Submit
- Select `Someone else` > Select a related contact > Continue to form > Fill out form > Submit
- Select `Someone else` > Select `Add new contact` > Continue to form > Fill out form > Submit
3. Navigate to [`membership/saacard`](https://deploy-preview-668--stanford-alumni.netlify.app/membership/saacard)
4. Login and verify that the correct card displays
5. Review code 👾 

# Associated Issues and/or People
- ADAPTSM-74
